### ISSUE TITLE
Add an option to show hover text in bottom bar like RCT1.

### ISSUE THREAD

**Issue by CybertronicToon** [OpenRCT2/OpenRCT2#3994]
It would be nice to have an option to show the hover text in a bar at the bottom like in RCT1. I find this looks nicer, and is easier to read than the RCT2 tooltips, as they show instantly, and they are always in one place, with. For those reasons, I prefer the RC1-style bottom text, rather than the RCT2-style tooltips, and would like to have it as an option to use. This is not a duplicate of #1423, as that wasn't about the text, it was just about the bar at the bottom being clickable to show recent messages. This is about the text, and not about it being clickable.

I suggest this bar to use the same style as the rest of the bottom toolbar, (transparency included).

**Comment by SijmenSchoon** [comment]
I could definitely see this being a useful feature for some

**Comment by Gymnasiast** [comment]
I consider it useless to be honest, as it takes up an incredible amount of space for little benefit.

**Comment by Horsti12** [comment]
I actuall didn't know that but when I now think about it than I like more the sytle from RCT1. It was easier for me at the beginning. Especiall it was better for the tutorial in rct1 compared to rct2.

**Comment by Patrik356b** [comment]
@Horsti12 has a good point

**Comment by Gymnasiast** [comment]
@Horsti12 Better for the tutorial? How? RCT2's tutorial was superior to RCT1's in my opinion as it contained annotations, which RCT1's lacked.

**Comment by Goddesen** [comment]
I think this would be a nice option to have. I occasionally find myself just looking at my park for a long while and zoning out - I have to move my mouse to some point where it won't accidentally hover over something so that I'm not distracted by tooltips!

**Comment by Patrik356b** [comment]
I have not tried the rct tutorial so I can't comment on that part

**Comment by CybertronicToon** [comment]
It would just be an option, you could disable it. You don't even have to enable it in any of the default themes, I would just like it as an option, as I find it is easier to read the text over there than fumbling around for the RCT2-style one that constantly movies, and I find the constant background it has is nice.

**Comment by Horsti12** [comment]
And when you add the bar at the bottom for notifications than you also could add the news button. As I already said in a other thread that I always open the news with the bar at the bottom of the screen in rct1.

**Comment by AaronVanGeffen** [comment]
@CybertronicToon Of course it could be optional, but bear in mind it does complicate the code base. Every instance that projects a tooltip on a viewpoint would have to project it to the notification area instead. This can probably be done in a maintainable way, but it is not as straightforward as you sketch.

I'm ambivalent to whether or not this option should be introduced, but it's obviously not a priority either way — focus right now is on finishing the implementation we have.

**Comment by CybertronicToon** [comment]
I mean, of course once the implementation of the tooltip code is done. Once the tooltip code is done, (I'm no expert in coding) could you possibly make it so there are two functions (or whatever) for it, one for if it is disabled, (current one) and one for when it is enabled?

**Comment by Gymnasiast** [comment]
Funny how it's always people who haven't contributed a single line of code telling us what to do and how easy something is.

If you only know Javascript, you're in for a nasty surprise if you try your hand at C. As for your 'optional' bit: every option complicates the code base. It's easy to demand more features if you're not the person who has to implement it - or worse, maintain it.

**Comment by jensj12** [comment]
Closed by #6172.
